src/pages/authentication/login/Login.css

```diff
@@ -0,0 +1,91 @@
+.form-wrapper {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    min-height: 100vh;
+   
+    
+  }
+  
+  .form-login {
+    width: 70%;
+    max-width: 40rem;
+    box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px;
+    padding: 2rem;
+    border-radius: 5px;
```

Comment:
u can consistently use relative units instead of absolute units.. But its not breaking change.